runelite-client/src/main/java/net/runelite/client/plugins/varrockmuseum/VarrockMuseumPlugin.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2018, Viktor <Viktor.horsmanheimo@teknik.io>
+ * Copyright (c) 2018, Tomas Slusny <slusnucky@gmail.com>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.varrockmuseum;
+
+import com.google.common.eventbus.Subscribe;
+import javax.inject.Inject;
+import net.runelite.api.Client;
+import net.runelite.api.events.WidgetLoaded;
+import net.runelite.api.widgets.Widget;
+import net.runelite.api.widgets.WidgetID;
+import net.runelite.api.widgets.WidgetInfo;
+import net.runelite.client.plugins.Plugin;
+import net.runelite.client.plugins.PluginDescriptor;
+
+@PluginDescriptor(name = "Varrock Museum Quiz")
+public class VarrockMuseumPlugin extends Plugin
+{
+	@Inject
+	private Client client;
+
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded widget)
+	{
+		if (widget.getGroupId() != WidgetID.VARROCK_MUSEUM_QUIZ_GROUP_ID)
+		{
+			return;
+		}
+
+		final Widget questionWidget = client.getWidget(WidgetInfo.VARROCK_MUSEUM_QUESTION);
+
+		if (questionWidget == null)
+		{
+			return;
+		}
+
+		final Widget answerWidget = VarrockMuseumAnswer.findCorrect(
+			client,
+			questionWidget.getText(),
+			WidgetInfo.VARROCK_MUSEUM_FIRST_ANSWER,
+			WidgetInfo.VARROCK_MUSEUM_SECOND_ANSWER,
+			WidgetInfo.VARROCK_MUSEUM_THIRD_ANSWER);
+
+		if (answerWidget != null && !answerWidget.getText().contains("<col="))
```

Comment:
Does the answer widget text contain any non-color tags? If not...

```
if (answerWidget == null)
{
    return;
}

String answerText = answerWidget.getText();

if (answerText.equals(Text.removeTags(answerText)))
{
    answerWidget.setText("<col=00FF80>" + answerText + "</col>");
}
```

Reply:
This was mostly to prevent inserting code tags twice in some random corner case, but by default it does not.